packages/psss-server/examples.http

```diff
@@ -0,0 +1,19 @@
+### /login
+POST http://localhost:8040/v1/login HTTP/2.0
+content-type: application/json
+
+{
+    "emailAddress": "username",
+    "password": "password"
+}
```

Comment:
We are going to add a `http.env` file soon to the reporting package so that devs can save credentials there without needing to worry about accidentally committing them to the repo (https://github.com/beyondessential/tupaia-backlog/issues/1662)
We will do it for psss-server there too!

Reply:
Ah awesome! Let me know when you've done that.